pyresample/geometry.py

```diff
@@ -1592,6 +1542,8 @@ def create_areas_def(self):
 
     def create_areas_def_legacy(self):
         """Create area definition in legacy format."""
+        warnings.warn("Pyresample's legacy areas file format is deprecated. "
```

Comment:
Should this be a `DeprecationWarning` like discussed earlier on Satpy.

Reply:
Probably, but the Satpy PR addressing this probably won't effect pyresample so I'm leaning towards leaving it and it can be changed in a separate PR.